### PR TITLE
travis, azure: installing OPS

### PR DESCRIPTION
**Pull request by btrb**
this PR is for installing OPS in travis

### CODE REVIEW

**Comment by mloubout** [comment]
Need to do it for pipelines too

**Comment by btrb** [comment]
> Need to do it for pipelines too

oki! i thought to reproduce what's done for yask.
although can't find references on it at azure-pipelines.yml  ..
any hints ?

File: Dockerfile.pipelines [hunk 1]
```diff
@@ -27,6 +27,12 @@ ENV CXX=g++-$gccVersion
 # Set Yask variables in case they are needed
 ENV YC_CXX=g++-$gccVersion
 
+# Set OPS variable in case they are needed
+ARG ops_compiler=gnu
+ARG ops_install_path=/tmp/OPS/ops
+ENV OPS_COMPILER=ops_compiler
+ENV OPS_INSTALL_PATH=ops_install_path
```
Comment (mloubout) [inline Dockerfile.pipelines]: $ missing

File: Dockerfile.pipelines [hunk 2]
```diff
@@ -129,6 +135,11 @@ RUN if [ $installWithPip == "true" ] ; then \
           make compiler-api ; \
           pip install -e . ; \
         popd ; \
+      elif [ $DEVITO_BACKEND == "ops" ] ; then \
+        git clone https://github.com/opesci/OPS.git /tmp/OPS ; \
+        pushd $OPS_INSTALL_PATH/c ; \
+          make ; \
+        popd ; \
```
Comment (mloubout) [inline Dockerfile.pipelines]: indentation

File: .travis.yml [hunk 3]
```diff
@@ -108,6 +108,14 @@ before_script:
       conda install swig; cd ../;
       git clone https://github.com/opesci/yask.git;
       cd yask; make compiler-api; pip install -e .; cd ../devito;
+    elif [[ $DEVITO_BACKEND == 'ops' ]]; then
+      pwd;
+      git clone https://github.com/opesci/OPS.git;
+      cd OPS/ops;
+      export OPS_INSTALL_PATH=$(pwd);
+      cd c;
+      make;
```
Comment (mloubout) [inline .travis.yml]: CC=gcc-7 make and should be gtg

Comment (btrb) [reply .travis.yml]: thank you, let me try that.
actually, the `make` seems to be working fine.. apparently the problem is that i'm not returning to the correct devito's installation path after "making"

i tried: `../../` , `$HOME` , `/` , etc...
now i'm trying `$HOME/opesci/devito`

Comment (mloubout) [reply .travis.yml]: `$HOME/opesci/devito` well that's not gonna work

Comment (btrb) [reply .travis.yml]: every trial takes me like 1,5h >.< 

glad you brought that..

let me try with:
```
      cd ../;
      git clone https://github.com/opesci/OPS.git;
      cd OPS/ops/c;
      CC=gcc-7 make;
      cd ../../../devito;
```

**Comment by btrb** [comment]
i'm closing this PR as it has been incorporated in #997

**Pull request closed without merging**